test/server/cards/02_SHD/units/KyloRenKillingThePast.spec.ts

```diff
@@ -89,6 +89,21 @@ describe('Kylo Ren, Killing the Past', function() {
                 expect(context.pykeSentinel.getPower()).toBe(2);
                 expect(context.pykeSentinel.getHp()).toBe(3);
             });
+
+            it('gives +2/0 and an Experience token to a non-Villainy/non-Heroism unit', function () {
```

Comment:
Thanks for taking care of this! Do you mind merging this test case into the above one? Can just reset his exhaust status and damage in between attacks and keep this test case name as a comment (see Rukh tests for kind of an example of how we stack multiple on-attack tests into one)